commisery/cli/__init__.py

```diff
@@ -259,4 +262,4 @@ def check(ctx, target):
 
 
 if __name__ == "__main__":
-    sys.exit(main())
+    sys.exit(main(None, None, None, None, None))
```

Comment:
Why is this list of `None` arguments suddenly necessary but not before? I don't see any code  changes that create such a need where it didn't exist before...

Reply:
Cleaning up the file with pylint errors ❌

Reply:
Will undo